src/components/Tool/FixMailingAddresses/FixMailingAddresses.test.tsx

```diff
@@ -198,6 +198,12 @@ describe('FixMailingAddresses', () => {
         expect(queryByTestId('loading')).not.toBeInTheDocument(),
       );
 
+      await waitFor(() =>
+        expect(
+          getByText('100 Lake Hart Drive, Orlando FL 32832'),
+        ).toBeInTheDocument(),
+      );
```

Comment:
You can use `findByText` here as well. Also, please can you help clean up this page and use a `findBy` wherever we can to improve the code?